lib/autoupdate/fips_enabled.go

```diff
@@ -0,0 +1,25 @@
+//go:build fips
```

Comment:
@zmb3 should I move this one to lib/modules to identify if the build supports the FIPS? Or there already a way to get this info from modules package

Reply:
I think this `modules.GetModules().IsBoringCrypto()` -- BoringCrypto is FIPS validated